README.md

```diff
@@ -37,3 +37,175 @@ docker run -it --rm \
   -v $(pwd)/boardswarm/share/server.conf:/etc/boardswarm/server.conf \
   ghcr.io/boardswarm/boardswarm:main
 ```
+
+## Manual Installation
+
+These instructions are for configuring a simple unencrypted setup with static
+JWKS based authentication.
+
+### Building
+
+Boardswarm needs rustc version 1.70 or newer, this rules out a number of stable
+distributions for now (the kind of thing you may want to run your
+infrastructure on). There are however a number of solutions
+
+ - Rather than installing the distribution version of Rust, use
+   [rustup](https://www.rust-lang.org/tools/install) to install the latest
+   version directly from the rust project.
+
+ - Build on a system that has the required version of rust installed and, as
+   rust builds static binaries, we can copy them to the target server once
+   built. (Note that rust will note the version of libc used during the build
+   and complain if the expected version isn't found, making this solution
+   unreliable at times.) For instance, assuming a Debian system:
+   ```
+   $ sudo apt install rustc
+   ```
+
+ - If installing software outside of the systems package management isn't for
+   you and building elsewhere isn't an option, then creating a container from
+   which to build boardswarm might be a solution. A simple Dockerfile to just
+   build boardswarm is as follows:
+   ```
+   FROM rust:slim-bookworm
+   ARG DEBIAN_FRONTEND=noninteractive
+
+   RUN apt-get update && \
+       apt-get install --yes \
+         pkg-config \
+         protobuf-compiler \
+         libudev-dev \
+         libssl-dev && \
+       rm -rf /var/lib/apt/lists/*
+
+   WORKDIR /usr/src/boardswarm
+
+   RUN cargo build \
+         --release
+   ```
+   Save this to the root of the git repos once we've checked it out below.
+
+If not building from a container (where this'll be handled by the container
+creation step), install required tools/libraries to build boardswarm. As we'll
+be pulling the latest from git, we'll need that installed too. Again, assuming
+a Debian based system:
+```
+$ sudo apt install rustc git pkg-config protobuf-compiler libudev-dev libssl-dev
+```
+
+Boardswarm is quite new and undergoing active development, so as previously
+mentioned, we'll build the latest from git. Clone the boardswarm repository:
+```
+$ git clone https://github.com/boardswarm/boardswarm.git
+```
+
+We can now build boardswarm:
+```
+$ cd boardswarm
+$ cargo build --release
+```
+Or, if wanting to use a container for this, run the following command:
+```
+podman build --volume=$(pwd):/usr/src/boardswarm .
+```
+
+### Server Install
+
+The "server" is the machine that will have the development boards attached to
```

Comment:
no need to have board attached per se ; it can purely act as a router as well (connecting to other boardswarms)

Reply:
Added note about use as router.